modin/pandas/io.py

```diff
@@ -54,15 +54,18 @@ def _read_parquet_pandas_on_ray(path, engine, columns, **kwargs):
             name for name in pf.metadata.schema.names if not PQ_INDEX_REGEX.match(name)
         ]
     num_splits = min(len(columns), RayBlockPartitions._compute_num_partitions())
-    # Each item in this list will be a column of original df
+    num_cpus = int(ray.global_state.available_resources()['CPU'])
+    # Each item in this list will be a list of column names of the original df
+    col_partitions = [columns[i:i + num_cpus] for i in range(0, len(columns), num_cpus)]
```

Comment:
I redid the math, and I think this will end up splitting the columns as evenly as possible:

```python
column_splits = len(columns) // num_cpus if len(columns) % num_cpus == 0 \
    else len(columns) // num_cpus + 1
col_partitions = [columns[i:i + column_splits] 
    for i in range(0, len(columns), column_splits)]
```

Reply:
I have seen the failure on Travis and not sure where it comes from, so would appreciate the fix
Regrading the code, silly mistake on my part. sorry about that.
Your code looks to be correct
Should I fix it and push?

Reply:
Sure! I will fix the travis error after you push the fix.